src/Builder/SplitItemBuilder.php

```diff
@@ -108,6 +129,14 @@ public function addSubMenu(string $text, \Closure $callback) : self
         $menu = $builder->build();
         $menu->setParent($this->menu);
 
+        $menu->checkboxStyle(function (CheckboxStyle $style) {
+            $style->fromArray($this->menu->getCheckboxStyle()->toArray());
+        });
```

Comment:
I would rather introduce `$menu->setCheckboxStyle()` and change this to `$menu->setCheckboxStyle(clone $this->menu->getCheckboxStyle())`